rdkit/Chem/EState/EState_VSA.py

```diff
@@ -41,7 +41,7 @@ def VSA_EState_(mol, bins=None, force=1):
   propContribs = EStateIndices_(mol, force=force)
   volContribs = VSAContribs_(mol)
 
-  ans = numpy.zeros(len(bins) + 1, numpy.float)
+  ans = numpy.zeros(len(bins) + 1, dtype='float')
```

Comment:
```
ans = numpy.zeros(len(bins) + 1, dtype=numpy.float)
```
No need to add the burden of interpreting the string.

Reply:
I don't agree with it. I changed it intentionally and got accepted in PR 4815. 
In particularly, this is a fix of `DeprecationWarning` in numpy 1.20 
Reference: https://numpy.org/devdocs/release/1.20.0-notes.html

Reply:
Ok, What about this?
```
ans = numpy.zeros(len(bins) + 1, dtype=float)
```

Reply:
> Ok, What about this?
> 
> ```
> ans = numpy.zeros(len(bins) + 1, dtype=float)
> ```

I am aware of that change but at the very least, I would prefer numpy.float64. 
According to PR 4815, he prefers a general solution to handle it.